RetakesPlugin/Modules/Managers/GameManager.cs

```diff
@@ -66,40 +69,34 @@ public void AddScore(CCSPlayerController player, int score)
 
     private int _consecutiveRoundsWon;
 
-    public void TerroristRoundWin()
+    private void TerroristRoundWin()
     {
         _consecutiveRoundsWon++;
 
-        if (_consecutiveRoundsWon == _consecutiveRoundWinsToScramble)
+        bool shouldScrambleNow = _isScrambleEnabled && _consecutiveRoundsWon == _consecutiveRoundWinsToScramble;
+        int roundsLeftToScramble = _consecutiveRoundWinsToScramble - _consecutiveRoundsWon;
+        // Almost scramble if 1-2 rounds left to automatic scramble
+        bool shouldAlmostScramble = _isScrambleEnabled && 0 < roundsLeftToScramble && roundsLeftToScramble <= 2;
```

Comment:
I'm not a fan of having the constant value on the left of an expression, I think this is what you're after instead:
```c#
var shouldAlmostScramble = _isScrambleEnabled && roundsLeftToScramble > 0 && roundsLeftToScramble <= 2;
```

Reply:
Ok will fix